src/status_im/contexts/centralized_metrics/events.cljs

```diff
@@ -17,7 +17,10 @@
   (when-let [event (tracking/tracked-event (interceptor/get-coeffect context :event))]
     (log/debug "tracking event" event)
     (when (push-event? (interceptor/get-coeffect context :db))
-      (native-module/add-centralized-metric event)))
+      (cond
+        (and (coll? event)
```

Comment:
@seanstrom, `coll?` will return true if `event` is a map, for example. Maybe here we should only use `vector?` to be extra precise?

Could be an `if` instead of `cond`. `doseq` is preferred to process with side-effects and ignore results.

```clojure
(if (vector? event)
  (doseq [e event]
    (native-module/add-centralized-metric e))
  (native-module/add-centralized-metric event))
```

Reply:
Ah good call for using `doseq` 👍 

I think you're right that we can remove the `coll?` check. Additionally, I think we can also support seqs by using `seq?` since we might return a sequence or vector from the tracking function, how does that sound?